TodoList/Tests/TodoList.Api.Tests/Helpers/ListItemComparer.cs

```diff
@@ -1,18 +1,18 @@
-﻿using System.Collections.Generic;
-using TodoList.Contracts.Models;
-
-namespace TodoList.Api.Tests.Helpers
-{
-    public class ListItemComparer : IEqualityComparer<ListItem>
-    {
-        public bool Equals(ListItem x, ListItem y)
-        {
-            return x.Id.Equals(y.Id) && x.Text.Equals(y.Text);
-        }
-
-        public int GetHashCode(ListItem obj)
-        {
-            return (obj.Id.GetHashCode() * 397) ^ (obj.Text?.GetHashCode() ?? 0);
-        }
-    }
+﻿using System.Collections.Generic;
+using TodoList.Contracts.Models;
+
+namespace TodoList.Api.Tests.Helpers
+{
+    internal class ListItemComparer : IEqualityComparer<ListItem>
```

Comment:
Consider going a step further and -- as this comparer will be used nowhere but in `EqualConstraintExtensions` -- moving it into the `EqualConstraintExtensions.cs` as a private class.